src/lib/openapi/spec/token-user-schema.ts

```diff
@@ -6,7 +6,7 @@ export const tokenUserSchema = {
     type: 'object',
     additionalProperties: false,
     description: 'A user identified by a token',
-    required: ['id', 'name', 'email', 'token', 'createdBy', 'role'],
+    required: ['id', 'email', 'token', 'createdBy', 'role'],
```

Comment:
The `name` property is missing in responses in pretty much all of our tests, so I'm guessing it's not there. Should it be?

In fact,  I don't know if it is ever used 🤔

Reply:
No, we definitely **do** use it sometimes: it's explicitly assigned to in UserService. I'm guessing that we do return it from the reset-password-controller as well if the user has a name.